Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/re3-pytorch/tracker/re3_vot_tracker.py", line 1, in <module>
    import re3_tracker
  File "/home/re3-pytorch/tracker/re3_tracker.py", line 3, in <module>
    import torch
  File "/usr/local/lib/python3.5/dist-packages/torch/__init__.py", line 84, in <module>
    from torch._C import *
RuntimeError: stoi
